var/spack/repos/builtin/packages/tasmanian/package.py

```diff
@@ -54,9 +55,10 @@ class Tasmanian(CMakePackage):
             description='CMake build type',
             values=('Debug', 'Release'))
 
-    depends_on('cmake@3.5.1:', type='build')
+    depends_on('cmake@3.5.1:', type='build', when='@:6.0')
+    depends_on('cmake@3.10:', type='build', when='@:7.0')
 
-    depends_on('python@2.7:', when='+python', type=('build', 'run'))
+    depends_on('python@3.5:', when='+python', type=('build', 'run'))
```

Comment:
Older versions should still be allowed to use Python 2.

Reply:
I'm not sure I did this right. All Tasmanian versions work with Python 2.7 and 3.5+, the problem is numpy.

When I have `python@2.7:` then latest numpy 1.17 doesn't work with 2.7 and spack gives an error.

Reply:
That's a bug in the concretizer, not a bug in your package. @tgamblin is working on fixing that, but in the meantime users will have to switch to Python 3 or specify which version of numpy to use by hand.

Reply:
Fair point, reverted the changes.